elide-standalone/README.md

```diff
@@ -36,23 +37,298 @@ To include `elide-standalone` into your project, add the single dependency:
+      @OneToMany(mappedBy = "artifact")
+      public List<ArtifactVersion> versions = new ArrayList<>();
+  }
+  ```
 
-ElideStandalone is configured by implementing the ElideStandaloneSettings interface. Please see the ElideStandaloneSettings class for documentation about fields.
+#### ArtifactVersion.java
 
-Similarly, if you need other metadata across your application, it is important to note that the injector is bound with the following:
+  ```java
+  @Include(type = "version")
+  @Entity
+  public class ArtifactVersion {
+      @Id
+      public String name = "";
 
-```java
-@Inject @Named("elideAllModels") Set<Class> entities;
+      public Date createdAt = new Date();
+
+      @ManyToOne
+      public ArtifactProduct artifact;
+  }
+  ```
+
+### Spin up the API
+
+So now we have some models, but without an API it is not very useful. Before we add the API component, we need to create the schema in the database that our models will use.   Our example uses liquibase to manage the schema.  When Heroku releases the application, our example will execute the [database migrations][https://github.com/aklish/elide-heroku-example/blob/master/src/main/resources/db/changelog/changelog.xml] to configure the database with some test data automatically.  This demo uses Postgres.  Feel free to modify the migration script if you are using a different database provider.
+
+There may be more tables in your database than models in your project.  Similarly, there may be more columns in a table than in a particular model.  Not only will our models work just fine, but we expect that models will normally expose only a subset of the fields present in the database. Elide is an ideal tool for building micro-services - each service in your system can expose only the slice of the database that it requires.
+
+### App & Settings
+
+Bringing life to our API is trivially easy. We need two new classes: Main and Settings.
+
+#### Main.java
+
+  ```java
+  public class Main {
+      public static void main(String[] args) throws Exception {
+          ElideStandalone app = new ElideStandalone(new Settings());
+          app.start();
+      }
+  }
+  ```
+
+#### Settings.java
+
+  ```java
+  public class Settings implements ElideStandaloneSettings {
+      @Override
+      public String getModelPackageName() {
+          return ArtifactGroup.class.getPackage().getName();
+      }
+
+      @Override
+      public Properties getDatabaseProperties() {
+          Properties options = new Properties();
+          
+          //Here we use H2 in memory instead of Postgres 
+          options.put("hibernate.dialect", "org.hibernate.dialect.H2Dialect");
+          options.put("javax.persistence.jdbc.driver", "org.h2.Driver");
+          options.put("javax.persistence.jdbc.url", "jdbc:h2:mem:db1;DB_CLOSE_DELAY=-1");
+          options.put("javax.persistence.jdbc.user", "sa");
+          options.put("javax.persistence.jdbc.password", "");
```

Comment:
Should we explain these configurations too? Or put a link to all references.

Reply:
👍